src/observer.ts

```diff
@@ -1,5 +1,36 @@
 import { Subscription } from './subscription';
 
+/** Symbol.observable addition */
+/* Note: This will add Symbol.observable globally for all TypeScript users,
+  however, we are no longer polyfilling Symbol.observable */
+declare global {
+  interface SymbolConstructor {
+    readonly observable: symbol;
+  }
+}
+
+/** Symbol.observable or a string "@@observable". Used for interop */
+//tslint:disable-next-line
+const { observable: Symbol_observable } = require('rxjs/symbol/observable');
```

Comment:
Why is this using an ES5 `require` and not an import? What if this is run in an ES6 environment?

Reply:
@felixfbecker you're right, that won't work in an esm-only env, my bad. This is pretty tricky:

* if I use `import { observable } from 'rxjs/_esm5/symbol/observable'`, then it breaks in node/cjs, since it's directly importing an esmodule file
* if I do `import { obserable } from 'rxjs/symbol/observable', then it breaks people in ESM-only envs (and closure-compiler, which is ESM-only-ish)

The real tricky part is we're on rxjs@5.5.11, both because 5 is still relatively widely adopted, and because our build scripts are using it and I don't have much time to sink into updating them for 6.x and pipe-all-the-things.

Reply:
I think we shouldn't rely on Rx symbol. We should have our own file that does the same - use `Symbol.observable` if available, otherwise use the string `"@@observable"`. And it should not mutate globals.